jetty-util/src/test/java/org/eclipse/jetty/util/BlockingArrayQueueTest.java

```diff
@@ -196,56 +198,55 @@ public void run()
 
         thread.start();
 
-        Thread.sleep(1000);
+        // Wait until the spawned thread is blocked in queue.take().
+        await().atMost(5, TimeUnit.SECONDS).until(() -> thread.getState() == Thread.State.WAITING);
```

Comment:
This is all very cool an all.... but do we have CQ to allow us to bring in this dependency?  Do we need CQs for tests???
Isn't it easier to write the 5 lines of code to do this spin and test rather than answer the CQ questions?

Reply:
I agree because I only used the most basic functionality of awaitility, so it was easy enough to substitute it with some helper.